transport/src/main/java/io/netty/channel/SingleThreadIoEventLoop.java

```diff
@@ -34,10 +35,10 @@
 public class SingleThreadIoEventLoop extends SingleThreadEventLoop implements IoEventLoop {
 
     // TODO: Is this a sensible default ?
-    protected static final int DEFAULT_MAX_TASKS_PER_RUN = Math.max(1,
-            SystemPropertyUtil.getInt("io.netty.eventLoop.maxTaskPerRun", 1024 * 4));
+    protected static final long DEFAULT_MAX_TASKS_PER_RUN_NS = TimeUnit.MILLISECONDS.toNanos(Math.max(10,
+            SystemPropertyUtil.getInt("io.netty.eventLoop.maxTaskProcessingDurationPerRun", 1000)));
```

Comment:
`io.netty.eventLoop.maxTaskProcessingDurationPerRunSoftLimit` what about

Reply:
@franz1981 @chrisvest WDYT ?

Reply:
Also @bryce-anderson

Reply:
In operating system lingo I think this is called a quantum, so it could also be `io.netty.eventLoop.maxTaskProcessingQuantumSoftLimit`, but He-Pin's suggestion is also fine.

Reply:
Can we include the units in the name of the property to avoid confusion and incorrect units? `io.netty.eventLoop.maxTaskProcessingQuantumNsSoftLimit`

Reply:
+1 to @Scottmitch's suggestion of including units in the name although the units look like their in ms and typically the units are at the end. I'd personally go with `io.netty.eventLoop.maxTaskProcessingQuantumMs` but don't have strong opinions other including the units.